growth.md

```diff
@@ -0,0 +1,54 @@
+
+We could think growth as a series of different metrics, but we focus on bringing in new users, expansion of use and brand awareness only. Our main goal is to get poeple to install the [GitHub App](https://github.com/marketplace/watermelon-context) and use it by sending PRs and updating them.
+
+Our current KPIs live in [Posthog](https://us.posthog.com/dashboard/84757), and every growth action should target that.
+
+## Goals
+### New Users
+
+For new users, we need to signup the whole team.
+
+This is honestly a very hard task, so we decided to focus on the managers, because, usually, they hold the power of decision, credit card and admin priviledges for repos.
+
+For this task we will get into their top of mind with more long form content such as blog posts, and will use their preferred social networks such as LinkedIn.
+
+### Expansion of Use
+
+Once we've managed to get the users to install, we need to get them addicted to the product. Regardless of how we charge the organization (per API call, packages of several uses, per seat...), we want them to expect the app on every PR.
+
+This is a team effort, making the product better all the time is key, but from the marketing side we can push the Individual Contributors in several ways. We can suggest curiosity, better ways to go about their flow, other tools, etc are all good ways, but distribution is key. We want them to see us when their brains are wandering, such as in X, TikTok or YouTube. We want to be a source of knowledge and fun.
+
+### Brand Awareness
+
+This takes the lowest priority in the ranking, being an afterthought even. We like people to remember our brand colors and logo, so we give out stickers and use bright clothes.
+We can push a few other things but don't fret about it.
+
+## Tactics
+We have to create content, hopefully quality content that trends. Remember each platform is different, and although 𝕏 allows for long form, it's probably not the best for every post, extend for all platforms.
+
+### Blog
+This is one of our favorite, as it's the one we control the most. To create a new blog post, [read our guide](https://github.com/watermelontools/public-handbook/blob/growth/BlogPosts.md), and to publish, got to [Webflow](https://watermelon-2023.design.webflow.com/). Now, to make it attractive, we use different platforms. It's all on the guide.
+
+One of the nicest things of writing blog posts is that we get SEO, increasing our score (currently around 35).
+
+The current voice of the blogs is serious, but you can still feel the author voice.
+
+### Newsletter 
+
+No current tactics other than the monthly update.
+
+### 𝕏
+This platform suffers from the brevity of it's reach. An amazing post will trend up a few hours, and can trend for any number of reasons. The current voice of the platform is kind of acid, usually mocking our team.
+
+### LinkedIn
+
```

Comment:
- The algorithm rewards silence for a while (apparently)
- Great for company announcements
- Ads are expensive and don't convert well
- Great idea to tag people
- Link on the coments